materialize-snowflake/snowflake.go

```diff
@@ -62,9 +62,6 @@ func schemasEqual(s1 string, s2 string) bool {
 }
 
 func (c tableConfig) Path() sql.TablePath {
-	if c.Schema == "" || schemasEqual(c.Schema, c.endpointSchema) {
```

Comment:
This bit is weird and cumbersome but I believe it is very important for backward compatibility: There was a time not so long ago that we did not allow resource-level schemas to even be set, see https://github.com/estuary/connectors/pull/980. Prior to this change, all resource paths were just the single `[]string{c.Table}`. If we were to start saying that all resource paths are now the 2-element `[]string{c.Schema, c.Table}`, it would change the connector's representation of pre-existing bindings' resource paths and would cause them to start over materializing from an empty checkpoint, which would not be good.